pwa/src/templates/templateParts/filters/verticalFilters/VerticalFiltersTemplate.tsx

```diff
@@ -415,10 +415,9 @@ export const VerticalFiltersTemplate: React.FC<VerticalFiltersTemplateProps> = (
                     onChange={() => setStatusRadioFilter(status.value)}
                     key={status.value}
                   >
-                    <input
-                      type="radio"
+                    <RadioButton
```

Comment:
The labels of the RadioButtons are not clickable anymore, which I think they should be. Also, theres no space between the radio button and the label.